local_development_setup.md

```diff
@@ -0,0 +1,245 @@
+# Local Development Notes
+
+This notes cover how you can setup a local development environment.
+
+## Setup Local Tools
+
+* Install [VSCode](https://code.visualstudio.com/) or another editor of choice
+
+* For SQL:
```

Comment:
@plooploops on local_development_setup.md I don't see where VS or VS with SSDT is used for SQL. Are the IDEs used or can we remove that part of the doc? If there is a place we need the IDE, can we also suggest installing SQL Server Management Studio as a VS alternative?

Reply:
Thank @corygstevenson  for taking a look.

Visual Studio is used for working with the [CDM](https://github.com/microsoft/OHDSIonAzure/blob/main/sql/cdm/v5.3.1/cdm_omop.sln).

You can use SSDT to compare [schema changes](https://docs.microsoft.com/en-us/sql/ssdt/how-to-use-schema-compare-to-compare-different-database-definitions?view=sql-server-ver15#:~:text=To%20compare%20database%20definitions%201%20On%20the%20SQL,created%20in%20the%20previous%20procedure.%20More%20items...%20) as part of CDM development.

I added in SQL Server Management Studio into the list for working with SQL.